app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("All friends should be vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")
+
+        if not visitor["wearing_a_mask"]:
```

Comment:
Using `get` with a default value ensures a safe check for the presence of the key "wearing_a_mask" in the dictionary, avoiding a `KeyError` and providing a default behavior when the key is absent.